tools/StaticAnalysis/Exceptions/Az.Storage/BreakingChangeIssues.csv

```diff
@@ -1,2 +1,83 @@
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet.StartAzureStorageFileCopyCommand","Start-AzStorageFileCopy","0","1050","The parameter set 'BlobInstanceFileInstance' for cmdlet 'Start-AzStorageFileCopy' has been removed.","Add parameter set 'BlobInstanceFileInstance' back to cmdlet 'Start-AzStorageFileCopy'."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet.StartAzureStorageFileCopyCommand","Start-AzStorageFileCopy","0","1050","The parameter set 'FileInstanceToFileInstance' for cmdlet 'Start-AzStorageFileCopy' has been removed.","Add parameter set 'FileInstanceToFileInstance' back to cmdlet 'Start-AzStorageFileCopy'."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet.StartAzureStorageFileCopyCommand","Start-AzStorageFileCopy","0","1050","The parameter set 'UriToFileInstance' for cmdlet 'Start-AzStorageFileCopy' has been removed.","Add parameter set 'UriToFileInstance' back to cmdlet 'Start-AzStorageFileCopy'."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet.StopAzureStorageFileCopyCommand","Stop-AzStorageFileCopy","0","2000","The cmdlet 'Stop-AzStorageFileCopy' no longer supports the parameter 'File' and no alias was found for the original parameter name.","Add the parameter 'File' back to the cmdlet 'Stop-AzStorageFileCopy', or add an alias to the original parameter name."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet.StopAzureStorageFileCopyCommand","Stop-AzStorageFileCopy","0","1050","The parameter set 'File' for cmdlet 'Stop-AzStorageFileCopy' has been removed.","Add parameter set 'File' back to cmdlet 'Stop-AzStorageFileCopy'."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet.GetAzureStorageShareStoredAccessPolicy","Get-AzStorageShareStoredAccessPolicy","0","1020","The cmdlet 'Get-AzStorageShareStoredAccessPolicy' no longer has output type 'Microsoft.Azure.Storage.File.SharedAccessFilePolicy'.","Make cmdlet 'Get-AzStorageShareStoredAccessPolicy' return type 'Microsoft.Azure.Storage.File.SharedAccessFilePolicy'."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet.GetAzureStorageBlobContentCommand","Get-AzStorageBlobContent","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Get-AzStorageBlobContent' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Get-AzStorageBlobContent'."
+"Az.Storage","Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet.GetAzureStorageBlobContentCommand","Get-AzStorageBlobContent","0","1050","The parameter set 'UriPipeline' for cmdlet 'Get-AzStorageBlobContent' has been removed.","Add parameter set 'UriPipeline' back to cmdlet 'Get-AzStorageBlobContent'."
```

Comment:
The exception in line 82-83 is from my change, others are from Yifan's change https://github.com/Azure/azure-powershell/pull/26594.
Just to make sure the PR won't take conflict.

Reply:
nice catch, let me include only your changes